### PR TITLE
wishlist added

### PR DESCRIPTION
**Pull request by aayushjain-code**

### CODE REVIEW

File: pages/Wishlist/wishlist.html [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+					<div class="btn-container">
+						<a href="#" class="btn"> Buy Now!</a>
+						<a href="#" class="btn-secondary"> Remove!</a>
+					</div>
+				</div>
+			</div>
+
+			<div class="box">
+				<div class="image">
+					<img src="/images/product-1.png" alt="">
```
Comment (MuskanChhatrasal) [inline pages/Wishlist/wishlist.html:80]: Add alt for images. 
Else Clean UI. Good for merge.

**Review by MuskanChhatrasal — APPROVED** [review]

File: pages/Wishlist/wishlist.html [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+				</div>
+			</div>
+
+			<div class="box">
+				<div class="image">
+					<img src="/images/product-2.png" alt="">
+				</div>
+				<div class="content">
+					<h2 class="card-title">Title</h2>
+					<div class="price"> 290.99 <span>350.99</span> </div>
```
Comment (ArjunJunna) [inline pages/Wishlist/wishlist.html:99]: You can use &#x20b9; for rupee symbol if you like...
<div class="price">&#x20b9; 290.99 <span>350.99</span> </div>
 Rest everything looks good.

**Review by ArjunJunna — APPROVED** [review]

File: pages/Wishlist/wishlist.html [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+				</div>
+			</div>
+
+			<div class="box">
+				<div class="image">
+					<img src="/images/product-6.png" alt="">
+				</div>
+				<div class="content">
+					<h2 class="card-title">Title</h2>
+					<div class="price"> 290.99 <span>350.99</span> </div>
```
Comment (MayankKumar10) [inline pages/Wishlist/wishlist.html:114]: you can use the del tag for line through texts insted text-decoration

**Review by MayankKumar10 — APPROVED** [review]
Everything else LGTM you can merge it

**Comment by aayushjain-code** [comment]
Applied the changes merging the branch

**Pull request merged**